draft-ietf-rtcweb-jsep.xml

```diff
@@ -741,66 +741,80 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
-          application can set an upscaling policy for each sent track.
-          For this case, if upscaling is permitted by policy, the
-          sender SHOULD apply upscaling in order to provide the desired
-          resolution. Otherwise, the sender MUST NOT apply upscaling.
-          The sender SHOULD NOT upscale in other cases, even if the
-          policy permits it. Upscaling MUST NOT change the track aspect
-          ratio.</t>
-
-          <t>If there is no appropriate and permitted scaling mechanism
-          that allows the received size limits to be satisfied, the
-          sender MUST NOT transmit the track.</t>
-
-          <t>If the attribute includes a "sar=" (sample aspect ratio)
-          value set to something other than "1.0", indicating the
-          receiver wants to receive non-square pixels, this cannot be
-          satisfied and the sender MUST NOT transmit the track.</t>
-
-          <t>In the special case of receiving a maximum resolution of
-          [0, 0], as described above, the sender MUST NOT transmit the
-          track.</t>
+          <t>This specification prescribes more specific behavior.
+          When a MediaStreamTrack, which is producing
+          video of a certain resolution (the "track resolution"), is attached
+          to a RtpSender, which is encoding the track video
+          at the same or lower resolution(s) (the "encoder resolutions"),
+          and a remote description is applied that references the sender and
+          contains valid "a=imageattr recv" attributes, it will
+          follow the rules below to ensure the sender does not transmit a
+          resolution that would exceed the size criteria specified in the
+          attributes. These rules are followed
+          as long as the attributes remain present in the remote description,
+          including if the track changes its resolution,
+          or if the track is replaced with a different track.
+          </t>
+
+          <t>Depending on how the RtpSender is configured, it may be producing
+          a single resolution, or, if simulcast <xref target="sec.simulcast" />
+          has been negotiated, multiple specific resolutions. In addition,
+          depending on the configuration, each encoding may have the
+          flexibility to reduce resolution when needed, or may be locked to
+          a specific output resolution.
+          </t>
+
+          <t>For each encoding being produced by the RtpSender, the following
+          rules are applied to determine what should be transmitted:
+            <list style="symbols">
+              <t>First, the right "a=imageattr recv" attribute is selected.
+              While JSEP endpoints will not include more than one such
+              attribute per media format, non-JSEP endpoints may send
+              multiple relevant "a=imageattr recv" attributes in a m=
+              section. The correct attribute is selected by taking the
```

Comment:
I would move this point about JSEP to the end of this graf and say:

"Note that JSEP endpoints will only include one "a=imageattr recv" attribute, but JSEP endpoints may receive SDP from non-JSEP endpoints which include  more than one such attribute."